src/main/java/bio/terra/dao/exception/FileSystemObjectAlreadyExistsException.java

```diff
@@ -0,0 +1,17 @@
+package bio.terra.dao.exception;
+
+import bio.terra.exception.BadRequestException;
+
+public class FileSystemObjectAlreadyExistsException extends BadRequestException {
```

Comment:
should we add a default constructor so deserialization will work? (same for all exception classes)